general/clients/codec-support.md

```diff
@@ -42,7 +42,7 @@ If the audio codec is unsupported or incompatible (such as playing a 5.1 channel
 
 ||Chrome|Firefox|Safari|Android|AndroidTV|iOS|Roku|Kodi|MPV Shim|
 |:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
-|FLAC|✅|❌|✅|✅||||✅|✅|
+|FLAC|✅|✅|✅|✅||||✅|✅|
```

Comment:
You should at least add a note here about that Firefox bug I sent you a while ago, as far as I know it's still open.